openpathsampling/tests/test_trajectory_transition_analysis.py

```diff
@@ -251,17 +248,17 @@ def test_analyze(self):
 
         self.analyzer.reset_analysis()
         self.analyzer.analyze([self.trajectory])
-        assert_equal(cont_frames[self.stateA].tolist(),
-                     self.analyzer.continuous_frames[self.stateA].tolist())
-        assert_equal(life_frames[self.stateA].tolist(),
-                     self.analyzer.lifetime_frames[self.stateA].tolist())
+        assert cont_frames[self.stateA].tolist() \
+                == self.analyzer.continuous_frames[self.stateA].tolist()
+        assert life_frames[self.stateA].tolist() \
+                == self.analyzer.lifetime_frames[self.stateA].tolist()
         A2B = (self.stateA, self.stateB)
-        assert_equal(trans_frames[A2B].tolist(),
-                     self.analyzer.transition_duration_frames[A2B].tolist())
-        assert_equal(cont_times[self.stateA].mean(),
-                     self.analyzer.continuous_times[self.stateA].mean())
-        assert_equal(life_times[self.stateA].mean(),
-                     self.analyzer.lifetimes[self.stateA].mean())
-        assert_equal(trans_times[A2B].mean(),
-                     self.analyzer.transition_duration[A2B].mean())
+        assert trans_frames[A2B].tolist() \
+                == self.analyzer.transition_duration_frames[A2B].tolist()
+        assert cont_times[self.stateA].mean() \
+                == self.analyzer.continuous_times[self.stateA].mean()
+        assert life_times[self.stateA].mean() \
+                == self.analyzer.lifetimes[self.stateA].mean()
+        assert trans_times[A2B].mean() \
+                == self.analyzer.transition_duration[A2B].mean()
```

Comment:
I prefer no backslash and with parenthesis instead (can't due suggestion due to he unaltered line)